### DISCUSSION TITLE
LiveChart2 build Error in UNO v5.3 with WinUI3

### DISCUSSION THREAD

**Discussion by nksharma1999** [unoplatform/uno#18126] [Q&A] [answered]
**LiveChart2 Installation Guide:** https://livecharts.dev/docs/UnoWinUi/2.0.0-rc2/Overview.Installation

I followed the documentation to install **LiveCharts2** in **UNO v5 with WinUI 3**, but I'm encountering a white screen on both **Android** and **Desktop**. When I check the output section, it shows the following error:

```
BUILD ERROR: Exception thrown: 'System.MissingMethodException' in System.Private.CoreLib.dll
Exception thrown: 'Microsoft.UI.Xaml.Markup.XamlParseException' in WinRT.Runtime.dll
WinRT information: Cannot create instance of type 'LiveChartsCore.SkiaSharpView.WinUI.PieChart' [Line: 36 Position: 9]
Uno.Extensions.Navigation.Navigators.FrameNavigator: Error: Show - Unable to navigate to page - The text associated with this error code could not be found.

Cannot create instance of type 'LiveChartsCore.SkiaSharpView.WinUI.PieChart' [Line: 36 Position: 9]
```

@jeromelaban  Does LiveCharts2 support UNO 5 with WinUI 3?

**Comment by jeromelaban** [comment]
Thanks for the question. Uno 5 supports LiveCharts, though there's one issue with WinAppSDK that we'll be releasing soon. You can try with the dev builds of uno.sdk to see if this fixes it for you.

**Reply by nksharma1999** [reply]
I am getting an error When I try to install dev builds of UNO.Sdk. I also tried multiple dev's builds.
<img width="387" alt="image" src="https://github.com/user-attachments/assets/cda7a2b9-f881-4929-becb-2255f3983f8b">

**Reply by jeromelaban** [reply]
Make sure to follow our docs for updating uno: https://platform.uno/docs/articles/upgrading-nuget-packages.html#projects-using-the-unosdk

**Comment by nksharma1999** [comment, marked as answer]
Closed Ticket Issues: https://github.com/unoplatform/uno/issues/18063
